Dockerfile

```
@@ -1,19 +1,16 @@
 # Development
-FROM golang:1.11.4-alpine AS development
-
-WORKDIR /go/src/github.com/tidepool-org/shoreline
-
-COPY . .
+FROM golang:1.12.7-alpine AS development
 
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
     apk add build-base git
 
-RUN go get -u github.com/golang/dep/cmd/dep
+ENV GO111MODULE on
 
-RUN dos2unix build.sh && \
-    dos2unix test.sh && \
-    ./build.sh
+WORKDIR /go/src/github.com/mdblp/shoreline
```

I don't see why we have to change all path in docker container.  Less changes, less conflicts in the future.
```suggestion
WORKDIR /go/src/github.com/tidepool-org/shoreline
```

---

When using GO111MODULE with github.com/tidepool-org/shoreline it fetch the tidepool project...

---

Just tested it locally, it doesn't fetch the github.com/tidepool-org/shoreline repository.

---

Also we don't need anymore the fix travis folder:

> `# fix travis working folder (see http://www.ruflin.com/2015/08/13/fix-for-travis-ci-failure-in-forked-golang-repositories/)`
which was present in the .travis.yml file.